raiden/network/transport/matrix.py

```diff
@@ -514,15 +558,15 @@ def _handle_message(self, room, event) -> bool:
                     "Can't parse message binary data",
                     message_data=data,
                     peer_address=pex(peer_address),
-                    exception=ex,
+                    _exc=ex,
```

Comment:
why was this changed? what is the point of this?

Reply:
`exception` is used internally by `structlog`. This threw a lot of warnings in the unit tests @LefterisJP created as part of #2300 . This fixes it by using a non-conflicting parameter name.

Reply:
This seems the wrong fix IMO, either use `log.exception` or [`log.<level>(exc_info=exc)`](http://www.structlog.org/en/stable/api.html#structlog.processors.format_exc_info)

Reply:
That will include the whole stacktrace of the exception, which isn't required here. This approach only logs the `repr` of the exception.